packages/auth0-server-js/EXAMPLES.md

```diff
@@ -563,6 +563,17 @@ await serverClient.loginBackchannel({
 });
 ```
 
+To obtain the tokens directly without establishing or updating a session, you can use the `getTokenByBackchannelAuth()` method. This is useful when CIBA is used out-of-band such as authorizing an AI agent to act on behalf of a user. The method accepts the same parameters as `loginBackchannel()` and returns the token response directly:
+
+```ts
+const tokenResponse = await serverClient.getTokenByBackchannelAuth({
+  bindingMessage: '',
+  loginHint: {
+    sub: 'auth0|123456789'
+  }
+});
+```
+
```

Comment:
Can we add a caveat message here that this will not store the token? As this can be confusing as all other getToken methods do store the tokens.